gelbooru-overhaul.user.css

```
@@ -0,0 +1,267 @@
+    display: block;
+}
+/* COLLAPSE SIDEBAR */
+/* FIX TAG CATEGORY HIDING ON NARROW */
+@media only screen and (max-width: 850px) {
+    .go-sm-unhidden {
+        display: inline;
+    }
+}
+/* COLLAPSE SIDEBAR */
+/* FIX SPACING ON NARROW */
+.go-mobile-unspacing {
+    margin-right: 0;
+}
+
+
+
+/* POST FIT TWEAKS */
+/* FIT VERTICALLY */
+.go-fit-height {
+    max-height: 95vh;
+    max-width: 95vw;
+    width: auto;
+    height: auto;
+}
+.go-fit-width {
+    max-width: 95vw;
+    max-height: none;
+    height: auto;
+    width: auto;
+}
+/* POST FIT TWEAKS */
+/* CENTER POST*/
+.go-center {
+    display: block;
+    margin: 0 auto;
+}
+/* POST FIT TWEAKS */
+/* CENTER NOTE CONTAINER */
+.note-container.go-center {
+    display: table;
+}
+/* POST FIT TWEAKS */
+/* FIT HORIZONTALLY ON NARROW */
+@media only screen and (max-width: 850px) {
+    .go-fit-width-on-narrow {
+        max-width: 90vw;
+        max-height: none;
+        height: auto;
+        width: auto;
+    }
+}
+/* POST FIT TWEAKS */
+/* ZOOM CURSORS */
+.go-cursor-zoom-in {
+    cursor: zoom-in;
+}
+.go-cursor-zoom-out {
+    cursor: zoom-out;
+}
+
+
+
+/* THUMBNAIL ENLARGE*/
+/* MAIN */
+.go-thumbnail-enlarge {
+    transform: scale(1);
+    transition: transform 169ms;
+    display: inline-block;
+}
+/* THUMBNAIL ENLARGE*/
+/* MAIN HOVER */
+.go-thumbnail-enlarge:hover {
+    transform: scale(var(--enlarge-scale, 2));
+    transition-delay: 142ms;
+    z-index: 420690;
+    position: relative;
+}
+/* THUMBNAIL ENLARGE*/
+/* CORNER RADIUS FOR THUMBNAILS */
+.go-thumbnail-corners {
+    border-radius: 3%;
+}
+
+/* THUMBNAIL ENLARGE*/
+/* LOADER CONTAINER */
+.go-loader {
+    position: relative;
+}
+/* THUMBNAIL ENLARGE*/
+/* LOADER MAIN */
+.go-loader:before {
+    content: "loading";
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 9999;
+    background: yellow;
+    color: red;
+    pointer-events: none;
+}
+
+
+
+/* CONFIG WINDOW */
+/* APPEARANCE */
+.go-config-window {
+    background: var(--background-color, white);
+    font-size: 1.2em;
+    box-shadow: 0 0 0.2rem 1px;
+}
+/* CONFIG WINDOW */
+/* LAYOUT */
+.go-config-window {
+    min-width: 45%;
+    min-height: 30%;
+    max-width: 80%;
+    max-height: 70%;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+}
+/* CONFIG WINDOW */
+/* POSITION */
+.go-config-window {
+    position: fixed;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 39013901;
+}
+/* CONFIG WINDOW */
+/* HIDING */
+.go-config-window-hidden {
+    visibility: hidden;
+}
+
+/* CONFIG WINDOW */
+/* HEADER */
+.go-config-window header a {
+    font-weight: bold;
+    font-size: 1.4em;
+    pointer-events: none;
+}
+/* CONFIG WINDOW */
+/* HEADER OVERFLOW FIX */
+.go-config-window header {
+    overflow: visible;
+}
+/* CONFIG WINDOW */
+/* FOOTER BUTTONS */
+.go-config-window footer input {
+    border: 1px solid;
+}
+
```

```suggestion
.go-config-window footer input:hover {
    background: grey;
    transition: 142ms;
    transition-timing-function: ease-out;
}
.go-config-window footer input:active {
    opacity: 0.8;
    transition: 42ms;
}

```
Add a little feedback to the config footer buttons.

---

`.go-config-window footer input:hover` causes a rather annoying artifact: the Close button is visible for a few ms after the window is closed
`.go-config-window footer input:active` alone seems to create the same effect but does not result in an artifact
Another option is to add a dissolve effect to the entire div.

---

it seems `:hover` is just invisible in dark mode due to gray
Since I'm trying to use the styles that are already on gelbooru, I would like to avoid another calculated color
I'll see what I can do about it later.
The buttons on gelbooru just aren't as beautiful.

---

remove fade-in is also a simple solution
```
transition: 142ms;
transition-timing-function: ease-out;
```

---

Done https://github.com/Enchoseon/gelbooru-overhaul-userscript/pull/2/commits/6745f5cf2da5f4e69640d36e56e465b59964b57b